test/unit-test/ota_cbor_utest.c

```diff
@@ -132,3 +134,224 @@ void test_OTA_CborDecodeStreamResponse()
         }
     }
 }
+
+void test_OTA_CborEncodeStreamRequest_Invalid()
+{
+    uint8_t cborWork[ CBOR_TEST_MESSAGE_BUFFER_SIZE ];
+    size_t encodedSize = 0;
+    uint32_t bitmap = CBOR_TEST_BITMAP_VALUE;
+
+    bool result = OTA_CBOR_Encode_GetStreamRequestMessage(
```

Comment:
Can you add a brief comment to each of the call/assert combinations that describes what makes the call invalid? Otherwise it's not very obvious unless you are familiar with the CBOR requirements.

Reply:
Added comments in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/150/commits/f6b16a4e84acc2710323a793bb8513a2811d7632